examples/select_cells.md

```diff
@@ -12,17 +12,14 @@
 
 `client.select_cells(where='gene', ...)`:
 ```python
->>> gene_symbol = client.select_genes().get_list()[0]['gene_symbol']
+>>> gene_symbol = client.select_genes(where="modality", has=["rna"]).get_list()[0]['gene_symbol']
 >>> cells_with_gene = client.select_cells(where='gene', has=[f'{gene_symbol} > 1'], genomic_modality='rna')
 >>> assert len(cells_with_gene) > 0
 
 >>> cells_with_gene_details_with_values = cells_with_gene.get_list(values_included=[gene_symbol])
 >>> cells_keys = cells_with_gene_details_with_values[0]['values'].keys()
 >>> assert list(cells_keys) == [gene_symbol]
 
->>> gene_symbol = client.select_genes().get_list()[10]['gene_symbol']
->>> cells_with_gene_atac = client.select_cells(where='gene', has=[gene_symbol], genomic_modality='atac')
```

Comment:
What has changed here? If `has` now needs to be an inequality, that seems reasonable, but if it's something else, please explain.